### ISSUE TITLE
Design of register and login page

### ISSUE THREAD

**Issue by abhisaini** [mentorient/mentorient#62]
Design of register and login page isnt attractive...we can try changing that

**Comment by m-henderson** [comment]
@abhisaini I agree! I am going to add the up-for-grabs and good first issue label.

**Comment by abhisaini** [comment]
well i can make a try...but as i am not much familier to c#.
So i can only make the design...
Then u may need to introduce the c# part in the forms.
Should i proceed?

**Comment by m-henderson** [comment]
@abhisaini that sounds good to me. for design, just stick to css and javascript. You can write all your css in this directory: wwwroot/css/site.css and all your JavaScript here: wwwroot/js/site.js
